azure-devops/cmake-configure-build.yml

```diff
@@ -27,6 +27,7 @@ steps:
     if exist "$(${{ parameters.buildOutputLocationVar }})" (
       rmdir /S /Q "$(${{ parameters.buildOutputLocationVar }})"
     )
+    set VSCMD_SKIP_SENDTELEMETRY=1

```

Comment:
Should we have a brief explanation of the issue being fixed here?

Reply:
Sure. I need to push changes to pick up Patch Tuesday so I can add comments too.

(There's probably a way to "bake in" this environment variable in provision-image although it's a bit more complicated)

Reply:
Yeah, I can bake it into `provision-image.ps1`. Update coming soon...